spyder/config/manager.py

```diff
@@ -226,5 +227,87 @@ def reset_to_defaults(self, section=None):
         config = self.get_active_conf(section)
         config.reset_to_defaults(section=section)
 
+    # Shortcut configuration management
+    # ------------------------------------------------------------------------
+    def _get_shortcut_config(self, context):
+        """
+        Return the shortcut configuration for global or plugin configs.
+
+        Context must be either '_' for global or the name of a plugin.
+        """
+        context = context.lower()
+        extra_valid_contexts = [

```

Comment:
I think we declare this variable as a constant on the top of this file, with the name `EXTRA_VALID_SHORTCUT_CONTEXTS`.

That way it'd be easier to spot it when adding a new extra context in the future.

Reply:
Sounds good